packages/adminrouter/extra/src/test-harness/tests/test_generic.config.yml

```diff
@@ -74,6 +74,15 @@ endpoint_tests:
     type:
       - master
   - tests:
+      is_upstream_req_ok:
+        enabled: true
```

Comment:
Good catch!

No, it is - it is just an artefact of rebasing/moving commits around. Fixed!